jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -344,15 +352,18 @@ YUI.add('juju-env-bakery', function(Y) {
     },
 
     /**
-      Interact to be able to sign-in to get authenticated.
-
-      @method _interact
-      @param {Function} The callback function to be sent in case of
-                        successful authentication
-      @param {Function} The callback function failure in case of
-                        wrong authentication
-      @return {Object} The asynchronous request instance.
-    */
+     Interact to be able to sign-in to get authenticated.
+
+     @method _interact
+     @param {Function} The callback function to be sent in case of
+            successful authentication.
+     @param {Function} The callback function failure in case of
+            wrong authentication.
+     @param {Function} The callback function failure in case of
+            wrong authentication.
+     @param {Object} e an xhr response object.
```

Comment:
Also I'd suggest a similar comment on _nonInteractiveVisitMethod below - it's not just any XHR response, but the particular one we're concerned with.